src/Stratis.Bitcoin/Utilities/RetryOptions.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+using System.Linq;
+
+namespace Stratis.Bitcoin.Utilities
+{
+    public class RetryOptions
+    {
+        public RetryOptions()
+        {
+            this.RetryCount = 1;
+            this.Delay = TimeSpan.FromMilliseconds(100);
+            this.ExceptionTypes = new[] { typeof(Exception) };
```

Comment:
Also this one needs to be empty

Reply:
Don't we want to retry on any exception by default? Otherwise, retry makes no sense as strategy won't do anything.

Reply:
No, only retry on what we know is a retriable error anything else needs to bubble up.

Reply:
I don't really have a strong opinion on either option. I changed it to be empty as per your comment.